### ISSUE TITLE
Connection not detected after return from outage

### ISSUE THREAD

**Issue by Aranad** [wireapp/wire-desktop#2047]
On linux desktop app, when the net connection drops, a message comes up correctly stating that there is no connection and that messages will not be send or received, but after the connection returns again Wire does not detect that its back and has to be restarted.

**Comment by Aranad** [comment]
The blue banner:
![screenshot from 2019-01-12 08-18-09](https://user-images.githubusercontent.com/1755922/51072122-b01f3a80-1642-11e9-9c27-522e29400d98.png)

Also I've just noticed that this only occurs for cable LAN connection, but the banner goes away propery when connected thru wifi.

**Comment by ffflorian** [comment]
@Aranad does this problem still occur?

**Comment by Aranad** [comment]
Yes, I just tried the new version released yesterday and the problem remains :-( It does seem slightly different though, because I can send messages even with the "no connection" banner present which I couldn't before - but I don't receive them wit the banner present.

Also, I've found that this problem only seems to occur with my satellite connection, regardless of whether I connect to the router with cable or Wifi. When I'm at a cafe or hotel which probably has DSL it works fine.

It's not only outages, it's also go in and out of standby that causes it.
